storage/source/StorageMetrics.cc

```diff
@@ -55,16 +91,58 @@ float StorageMetrics::recoverySpeed() {
   return 0.0;
 }
 
-float StorageMetrics::degenerateRead() {
-  // TODO(ejd6): implement
-  return 0.0;
+float StorageMetrics::degenerateRead(
+    std::string file,
+    std::string destination,
+    std::vector<std::pair<std::string, std::string>> targetDatanodes) {
+  // Timing should be done on verbosity level 9.
+  el::Loggers::setVerboseLevel(9);
```

Comment:
Should we use a `const` variable to define the timing verbosity level? (cc @cannon10100) It would avoid this magic number.

Reply:
We could certainly define it as a macro or global const. Currently we only expect this to be set during testing, so it wasn't seen as a significant magic number.